tensorflow/lite/toco/tflite/operator.cc

```diff
@@ -182,6 +182,22 @@ class DepthwiseConvolution
   }
 };
 
+class Abs : public SimpleOperator<FloorDivOperator> {
+ public:
+  explicit Abs() : SimpleOperator("ABS", OperatorType::kAbs) {}
+  int GetVersion(const OperatorSignature& op_signature) const override {
+    const string& input_name = op_signature.op->inputs[0];
+    const Array& input_array = op_signature.model->GetArray(input_name);
+    // Version 2 supports signed/unsigned int8 and signed int32 input types.
+    if (input_array.data_type == ArrayDataType::kInt8 ||
+        input_array.data_type == ArrayDataType::kUint8 ||
+        input_array.data_type == ArrayDataType::kInt32) {
```

Comment:
thanks! please update version in op resolver too https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/register.cc#L161

Reply:
Sure. I am thinking that do we need a guide to let people know the *must* of adding operators related code to TFLite? I believe many people have similar questions like  https://github.com/tensorflow/tensorflow/issues/31879